### PR TITLE
system: Remove double scrollbar on the page

### PR DESCRIPTION
**Pull request by garrett**
Make #journal element (immediate child to body) stretch to fill <body>

### CODE REVIEW

File: pkg/systemd/logs.scss [hunk 1]
```diff
@@ -2,6 +2,11 @@
 @import "../lib/journal.css";
 @import "./system-global.scss";
 
+body {
+    // Ensure journal UI fills the entire page (and does not run over)
+    display: grid;
+}
+
 #journal {
     display: grid;
```
Comment (martinpitt) [inline pkg/systemd/logs.scss:11]: Do we still need this as well then? (Intuitively, `display:grid` should be inherited by all `<body>` children?)

**Review by martinpitt — APPROVED** [review]
Eww, that was ugly indeed, thanks for spotting and fixing! Just one question, but good to land for sure.

File: pkg/systemd/logs.scss (hunk 1 above)
Comment (garrett) [reply pkg/systemd/logs.scss:11]: Display: grid is never inherited. Immediate children participate in the grid, certainly, but are not grid elements themselves. 

#journal is a grid (with children specified in different ways) within a grid that only has one cell (to stretch the child to the content, which means the page, in this case).

**Comment by garrett** [comment]
I still think the best option would be to merge the #journal element into body, but there might be other side effects. (Unsure. There might also not be.)

**Review by marusak — APPROVED** [review]
Nice, thanks!

Works nicely here.

**Comment by martinpitt** [comment]
`TestMachines.testExternalConsole` now fails everywhere. Obviously unrelated to this PR, but @KKoukiou that sounds like some regression crept into master. At first sight this looks like your consoles PR collided with @marusak 's "rename wait_present()" PR, and the console test's `wait_present()` needs to be at least renamed, or better fixed to `wait_visible`?

**Comment by garrett** [comment]
On IRC, @KKoukiou mentioned she doesn't like this fix as it's a local one and other pages may run into it.

But I responded that I'm not sure if there's a universal fix, unless the page layout is done in the body.

Unless you're using a document or have a page that acts like a document, then you _must_ override `<body>`'s default setting to keep flowing and scroll. And that's what we're doing here; telling body to pull iits children into a grid to align to the page.

**Comment by KKoukiou** [comment]
> (BTW: I do agree with @KKoukiou that we should not have one-offs for each page whenever possible. I'm not sure if there's something we can do Cockpit-global without having the layout class (or ID, which really should be a class for CSS though) in the body.)

Let's merge this one, and if we run into this in another page let's consider it again if this can be generalized.

**Pull request merged**